source/features/file-finder-buffer.tsx

```diff
@@ -8,8 +8,9 @@ import features from '.';
 const getBufferField = onetime((): HTMLInputElement => (
 	<input
 		type="text"
-		className="form-control tree-finder-input p-0 ml-1 border-0"
-		style={{marginTop: '-0.19em'}}
+		className="form-control tree-finder-input p-0 border-0"
+		style={{marginTop: '-0.35em', backgroundColor: 'transparent'}}
```

Comment:
Alignment was not an issue so don't change that.
```suggestion
		className="form-control tree-finder-input p-0 border-0"
		style={{marginTop: '-0.19em', backgroundColor: 'transparent'}}
```

This is now misaligned: 
<img width="565" alt="" src="https://user-images.githubusercontent.com/1402241/93495981-53bef480-f8d4-11ea-832c-bebb3c31ac46.png">

Reply:
`0.35em` is the right alignment for me. Could you please try `0.19em` with `  vertical-align: baseline`?